source/Locales/Compactness.lagda

```diff
@@ -60,6 +64,20 @@ We also define the type `𝒦 X` expressing the type of compact opens of a local
 𝒦 : Locale 𝓤 𝓥 𝓦 → 𝓤 ⊔ 𝓥 ⊔ 𝓦 ⁺  ̇
 𝒦 X = Σ U ꞉ ⟨ 𝒪 X ⟩ , is-compact-open X U holds
 
+𝒦-is-set : (X : Locale 𝓤 𝓥 𝓦) → is-set (𝒦 X)
+𝒦-is-set X {(K₁ , κ₁)} {(K₂ , κ₂)} =
+ Σ-is-set
+  carrier-of-[ poset-of (𝒪 X) ]-is-set
+  (λ U → props-are-sets (holds-is-prop (is-compact-open X U)))
+
+𝒦-equality : (X : Locale 𝓤 𝓥 𝓦) {K₁ K₂ : ⟨ 𝒪 X ⟩}
```

Comment:
Probably `to-𝒦-＝` follows better the unspoken convention we have been using.

Reply:
Done in commit fd67b01084cc1c354884a96260527180905292f5 on `master`.